content/blogs/2023-11-08-when-use-kestra-airbyte-fivetran.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "When to choose Airbyte/Fivetran or Kestra for data ingestion ?"
+description: "Learn whenever it's best to use Kestra, Airbyte or Fivetran for data ingestion, or when it's good to use a mix of them"
+date: 2023-11-08T08:00:00
+category: Solutions
+author:
+  name: Benoit Pimpaud
+  image: "bpimpaud"
+image: /blogs/2023-11-08-when-use-kestra-airbyte-fivetran.jpg
+---
+
+As the data stack is maturing, it’s good to ask ourselves what are the overlaps of the tools we have at hand.
+At Kestra we often hear this question: “is it better to use Airbyte/Fivetran or Kestra for data source ingestion?”
+
+It’s a very good question. Airbyte and Fivetran are designed to move data from source to destination, and Kestra also has similar features.
+
+In this blog we will detail where each tool shines and when they can fit together.
+
+## Airbyte/Fivetran: moving data from source to destination
+
+Airbyte or Fivetran simplify the process of moving data from source systems to desired destinations. It comes with many connectors such as databases, APIs or file systems.
+
+Thanks to Airbyte Cloud, Airbyte Terraform provider or the Airbyte CLI, it’s easy to use it whatever the stack you have. Whatever practice strategies you use. With more than 300 pre-built connectors, Airbyte’s catalog is the largest in the industry - supported by an open-source community.
```

Comment:
```suggestion
Thanks to Airbyte Cloud, Airbyte Terraform provider or the Airbyte CLI, it’s easy to use it whatever the stack you have. Whatever practice strategies you use. With more than 300 pre-built connectors, Airbyte’s catalog is the largest in the industry - supported by an open-source community.
```

some sentences are broken in an unexpected way e.g. "Whatever practice strategies you use." is not a full sentence